src/main/java/at/bitfire/ical4android/JtxICalObject.kt

```diff
@@ -1211,8 +1211,10 @@ duration?.let(props::add)
             )
         }
 
-
         this.relatedTo.forEach { related ->
+            if(related.text == this.uid)                // ignoring entries that are linked to itself (issue with Tasks.org)
+                return@forEach
+
```

Comment:
I haven't heard about this before, do you know the steps to reproduce this in Tasks.org?

Reply:
Hello @abaker , there was an issue that was hard to reproduce. This was an attempt to resolve the problem. However, I found another place where the actual problem originates, so nothing to worry for Tasks.org for now ;-) I will delete this PR